packages/lib/utils/joplinCloud/index.ts

```diff
@@ -405,7 +405,7 @@ export function getPlans(stripeConfig: StripePublicConfig): Record<PlanName, Pla
 			featureLabelsOff: getFeatureLabelsByPlan(PlanName.Teams, false),
 			cfaLabel: _('Try it now'),
 			cfaUrl: '',
-			footnote: _('Per user. Minimum of %d users.', 2),
+			footnote: _n('Per user. Minimum of %d user.', 'Per user. Minimum of %d users.', 2, 2),
```

Comment:
No need to make it too complicated here - it's unlikely that will change

```suggestion
			footnote: _('Per user. Minimum of 2 users.'),
```